app/api/sync/syncWorker.js

```diff
@@ -29,12 +29,12 @@ export default {
       await prev;
 
       if (change.deleted) {
-        return synchronizer.syncData(url, 'delete', change, { _id: change.mongoId });
+        return synchronizer.syncData(url, name, 'delete', change, { _id: change.mongoId });
       }
 
       const data = await config.shouldSync(change);
       if (data) {
-        return synchronizer.syncData(url, 'post', change, data, lastSync);
+        return synchronizer.syncData(url, name, 'post', change, data, lastSync);
```

Comment:
@RafaPolit i think syncData already had too many arguments before, now five, i think the name can be easily removed if we move the responsibility of updating the lastSync from syncData here after each syncData call, what do you think.

Reply:
My first hunch was to use the URL as the unique identifying property, but thought name was more semantic.
Still, yes, I can look into who is responsible for what and who needs to know which values.

Reply:
I have moved them upwards as you suggested.  Still, the function had 5 args already, so I combined 3 those that 'make sense' to be a data object to an object, and left the other two as options.

I'm not really kin of "masking" the "too many args" into objects, but, in this case, it kind of made sense.  What do you think?

Reply:
looks good 👍, not sure about the object either but definitely better.